anydex/test/wallet/stellar/test_stellar_db.py

```diff
@@ -0,0 +1,116 @@
+import os
+from datetime import datetime
+
+from sqlalchemy.orm import session as db_session
+
+from anydex.wallet.stellar.xlm_db import StellarDb, Secret, Transaction
+from anydex.test.base import AbstractServer
+
+
+class TestStellarDb(AbstractServer):
+    def setUp(self):
+        super().setUp()
+        self.db = StellarDb(os.path.join(self.session_base_dir, 'stellar.db'))
+        self.tx = Transaction(hash='96ad71731b1b46fceb0f1c32adbcc16a93cefad1e6eb167efe8a8c8e4e0cbb98',
+                              ledger_nr=26529414,
+                              date_time=datetime.fromisoformat('2020-06-05T08:45:33'),
+                              source_account='GDQWI6FKB72DPOJE4CGYCFQZKRPQQIOYXRMZ5KEVGXMG6UUTGJMBCASH',
+                              operation_count=1,
+                              transaction_envelope="AAAAAOFkeKoP9De5JOCNgRYZVF8IIdi8WZ6olTXYb1KTMlgRAAAAZAGOO"
```

Comment:
PEP8 🐸 Code seems to be going over 120 lines. If desired, probably can be fixed just by putting all transaction attributes into new line, just intended by one tab, or by reformatting _transaction_envelope_ attribute.